docs/helpers/common.json

```diff
@@ -20,5 +20,49 @@
       "ara":[
         "أبانا الذي في السموات، ليتقدس إسمك. ليأت ملكوتك. لتكن مشيئتك، كما في السماء كذلك على الأرض. خبزنا كفافنا أعطنا اليوم. وأغفر لنا ذنوبنا، كما نغفر نحن أيضاً للمذنبين إلينا. ولا تدخلنا في تجربة، لكن نجنا من الشرير. بالمسيح يسوع ربنا، لأن لك الملك والقوة والمجد إلى الأبد. آمي"
       ]
+    },
```

Comment:
I think this should all be combined

Reply:
I was thinking about that but if its put all together how would we separate it by each role?

Reply:
Hmm, good point. However, I initially implemented the helpers to act as macro system. So I guess in this case these helpers are used often enough justify putting them in there but in most cases they are all together so maybe we should look into stuffing them into a separate document. We don't want to use the helpers as piecemeal system.

Reply:
Got it.. I frequently had to write these so I though I would just put them in the helpers to expedite the process.